### DISCUSSION TITLE
How to decrease the space between title and toc in Quarto

### DISCUSSION THREAD

**Discussion by Theaang** [quarto-dev/quarto-cli#6495] [Q&A]
### Description

I am using Quarto to create a PDF. When I chose not to include the author and date, the distance between the title and table of contents did not shorten but left a vertical space. I have attempted modifications using LaTeX methods but did not work. Here is the original setting:
```
---
title: "Supplemental Material 3: Preprocessed Effect Sizes from Invididual Source Data"
toc: true
format: pdf
editor: visual
---
```
<img width="672" alt="quarto question" src="https://github.com/quarto-dev/quarto-cli/assets/124239794/c6ad55bf-008b-4c58-865b-8860cb586986">

I would greatly appreciate any help on this question!

**Comment by mcanouil** [comment]
As PDF format uses LaTeX you have to use LaTeX code to do that.
The spacing is the default.

One way (possibly not the best) is to use the partials provided by Quarto (<https://quarto.org/docs/journals/templates.html#html-partials>) and add a line before the toc is added `\vspace{-2cm}`.

You can search for a better solution on the Internet and use `keep-tex: true` to fiddle with LaTeX code.

**Reply by sebacea** [reply]
specific suggestion to incorporate in a `.qmd` document after the yaml
```latex

```{=latex}
\vspace{-2.5cm}```
```
